src/swell/configuration/jedi/interfaces/geos_marine/observations/adt_swot_nadir.yaml

```
@@ -0,0 +1,70 @@
+obs space:
+  name: adt_swot_nadir
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: '{{cycle_dir}}/adt_swot_nadir.{{window_begin}}.nc4'
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: '{{cycle_dir}}/{{experiment_id}}.adt_swot_nadir.{{window_begin}}.nc4'
+  simulated variables: [absoluteDynamicTopography]
+obs operator:
+  name: ADT
+obs error:
+  covariance model: diagonal
+obs filters:
+- filter: Domain Check
+  where:
+  - variable: {name: GeoVaLs/sea_area_fraction}
+    minvalue: 0.9
+- filter: Domain Check
+  where:
+  - variable: { name: GeoVaLs/sea_surface_temperature}
+    minvalue: 15.0
```

```suggestion
    minvalue: 7.5
```

i dropped this to 7.5 in other ADTs following @ehackert's advice, we can see how that works.